frontend/openchat-shared/src/domain/chit.ts

```diff
@@ -23,71 +23,74 @@ export type MemeContestWinner = {
 };
 
 export const achievements = [
-    "appointed_group_mod",
-    "direct_chat_5",
-    "changed_theme",
-    "appointed_as_group_mod",
-    "favourited_chat",
-    "appointed_group_admin",
-    "received_reaction",
-    "voted_on_poll",
-    "appointed_as_group_admin",
-    "own_group_with_100_diamond_members",
+    "streak_3",
+    "streak_7",
+    "streak_14",
+    "streak_30",
+    "set_bio",
+    "set_avatar",
+    "joined_group",
+    "joined_community",
+    "sent_direct_message",
+    "received_direct_message",
+    "upgraded_to_diamond",
+    "set_display_name",
+    "upgrade_to_gold_diamond",
+
+    "sent_text", // SUCCEEDED
+    "sent_image", // SUCCEEDED
+    "sent_file", // SUCCEEDED
+    "quote_replied", // SUCCEEDED
+    "replied_in_thread", // FAILED
+    "sent_poll",
+    "sent_crypto",
+    "sent_video",
+    "sent_audio",
+    "sent_giphy",
+    "sent_prize",
+    "sent_meme",
+    "sent_reminder",
+    "sent_swap_offer",
+    "started_call",
+    "edited_message",
+    "reacted_to_message",
+    "forwarded_message",
+    "deleted_message",
+    "tipped_message",
+
+    "proved_unique_personhood",
     "set_community_display_name",
     "appointed_group_owner",
-    "own_group_with_10_diamond_members",
     "joined_gated_group_or_community",
-    "started_call",
     "appointed_as_group_owner",
-    "tipped_message",
-    "sent_giphy",
     "set_community_access_gate",
     "had_message_tipped",
     "swapped_from_wallet",
-    "edited_message",
-    "reacted_to_message",
     "accepted_swap_offer",
     "joined_call",
-    "sent_image",
     "enabled_disappearing_messages",
-    "forwarded_message",
-    "sent_prize",
     "followed_thread",
-    "sent_reminder",
     "proved_unique_personhood",
-    "deleted_messge",
-    "sent_text",
-    "owned_group_with_1000_diamond_members",
-    "sent_swap_offer",
-    "quote_replied",
-    "owned_group_with_1_diamond_member",
-    "sent_crypto",
     "pinned_message",
     "received_crypto",
     "translation_accepted",
-    "replied_in_thread",
-    "direct_chats_10",
-    "direct_chats_20",
     "set_group_access_gate",
-    "sent_file",
-    "sent_meme",
-    "sent_poll",
-    "sent_audio",
     "suggested_translation",
-    "sent_video",
-    "streak_3",
-    "streak_7",
-    "streak_14",
-    "streak_30",
-    "set_bio",
-    "set_avatar",
-    "joined_group",
-    "joined_community",
-    "sent_direct_message",
-    "received_direct_message",
-    "upgraded_to_diamond",
-    "set_display_name",
-    "upgrade_to_gold_diamond",
+    "appointed_group_mod",
+    "changed_theme",
```

Comment:
Curious about `changed_theme` - theme is device specific and is not recorded on the back end at all. So how are we able to track that?